### ISSUE TITLE
[FEATURE REQUEST] Ability to blacklist/pin packages from the uninstall section

### ISSUE THREAD

**Issue by ppvnf** [marticliment/WingetUI#889]
## Before submitting the issue

- [X] **I have checked that the requested feature is not a duplicate or, if it is, it needs to be reopened**
- [X] **I have checked that I am running the latest version of WingetUI**
- [X] **I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section**
- [X] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#feature-requests) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**

## Is your feature request related to a problem? Please describe.

This feature would be useful specially if packages are automatically being updated and you want to keep some in a specific version

## Describe the solution you'd like

Add the ability to blacklist/pin packages from the uninstall section (maybe in the context menu)

related to https://github.com/marticliment/WingetUI/issues/718 and https://github.com/marticliment/WingetUI/issues/284

**Comment by marticliment** [comment]
you mean like backlisitng them from the uninstall section, right?

**Comment by ppvnf** [comment]
yes

**Comment by ppvnf** [comment]
<img width="552" alt="Screenshot 2023-04-20 170134" src="https://user-images.githubusercontent.com/73800734/233407007-5ea34391-ff16-4c13-b2aa-3ebb8af2e693.png">
When you added the ignore update option in https://github.com/marticliment/WingetUI/issues/902 it was also added to the uninstall section and update section (I presume). However, in the uninstall section it should be possible to just check the box to blacklist (without actually clicking the uninstall button). Also, it would be nice to be able to have the checkbox already checked for the packages already blacklisted, so you could unblock them more easily

**Comment by ppvnf** [comment]
@marticliment

**Comment by marticliment** [comment]
the blacklist feature is going go have a full redesign, in which those things will.be added. could you please list them on a new issue please?
